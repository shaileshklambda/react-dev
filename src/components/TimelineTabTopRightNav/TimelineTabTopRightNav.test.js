import React from "react";
import renderer from "react-test-renderer";
import Timelinetabtoprightnav from ".";

it("Timelinetabtoprightnav: default", () => {
  const component = renderer.create(<Timelinetabtoprightnav />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
