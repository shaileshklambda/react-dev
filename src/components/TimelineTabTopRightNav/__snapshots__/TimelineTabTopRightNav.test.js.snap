// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timelinetabtoprightnav: default 1`] = `
<ul
  className="topRightNav nav nav-pills pull-right"
  role={null}
>
  <li
    className="active"
    role="presentation"
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      title="Test"
    >
      <span
        className="dotIcon active"
      />
      <span
        className="status fontLato"
      >
        5/5
      </span>
    </a>
  </li>
  <li
    className=""
    role="presentation"
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      title="Test"
    >
      <span
        className="dotIcon inProgress"
      />
      <span
        className="status fontLato"
      >
        2/5
      </span>
    </a>
  </li>
  <li
    className="iconGroup"
    role="presentation"
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      title="Access Key"
    >
      <div>
        <img
          className=""
          height={20}
          onClick={[Function]}
          src={
            Object {
              "ReactComponent": [Function],
              "__esModule": true,
              "default": "access-key.svg",
            }
          }
          width={25}
        />
      </div>
    </a>
  </li>
  <li
    className=""
    role="presentation"
  >
    <a
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      title="Help"
    >
      <div>
        <img
          className=""
          height={20}
          onClick={[Function]}
          src={
            Object {
              "ReactComponent": [Function],
              "__esModule": true,
              "default": "help-center.svg",
            }
          }
          width={25}
        />
      </div>
    </a>
  </li>
</ul>
`;
